src/main/kotlin/com/gamenews/data/ArticlesDatabase.kt

```diff
@@ -0,0 +1,63 @@
+package com.gamenews.data
+
+import com.gamenews.models.Article
+import org.litote.kmongo.coroutine.coroutine
+import org.litote.kmongo.reactivestreams.KMongo
+
+/**
+ * Class for our database queries
+ */
+class ArticlesDatabase {
+    private val client = KMongo.createClient().coroutine
+    private val database = client.getDatabase("ArticlesDatabase")
```

Comment:
This database should also be passed as an argument to the `ArticleDatabase`. That will making it easier to test.

Reply:
Something like:
```
class ArticlesDatabase(
    private val database: Database<Article>,
) {...} 
```

Reply:
Done